Leanplum/Cache/LPCache.m

```diff
@@ -41,5 +41,68 @@ - (void)setFileAttributes:(NSArray<LPFileAttribute *> *)fileAttributes {
     return regions;
 }
 
+- (NSArray<LPEventRule *> *)loadEventRules {
+    return nil;
```

Comment:
these are placeholders? should we add `//TODO`

Reply:
yea